README.md

```diff
@@ -242,7 +242,7 @@
 
 [Instapaper](https://web.archive.org/web/20211124170124/https://medium.com/making-instapaper/instapaper-outage-cause-recovery-3c32a7e9cc5f). Also [this](https://blog.instapaper.com/post/157027537441). Limits were hit for a hosted database. It took many hours to migrate over to a new database.
 
-[Intel](https://42gems.com/blog/?p=735). A scripting bug caused the generation of the divider logic in the Pentium to very occasionally produce incorrect results. The bug wasn't caught in testing because of an incorrect assumption in a proof of correctness.
+[Intel](https://42gems.com/blog/?p=735). A scripting bug caused the generation of the divider logic in the Pentium to very occasionally produce incorrect results. The bug wasn't caught in testing because of an incorrect assumption in a proof of correctness. (FDIV bug, 1994.)
```

Comment:
What's this comment?

Reply:
Oh, yes, that's not a link-fix, sorry.  Added the comment for clarity, as the FDIV bug is a famous case and the entry ought to not that. Likewise the date - this is not a recent Intel problem.  We could even link to Wikipedia. https://en.wikipedia.org/wiki/Pentium_FDIV_bug

Reply:
The parenthetical doesn't add much for me, but if you added something like `See [the Wikipedia article on 1994 FDIV bug](https://en.wikipedia.org/wiki/Pentium_FDIV_bug) for more information.`

Reply:
Good thinking, I've made that edit (hopefully in a way which makes it part of this pull request)